easybuild/easyconfigs/c/cget/cget-0.2.0-GCCcore-11.2.0.eb

```
@@ -0,0 +1,37 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+easyblock = 'PythonPackage'
+
+name = 'cget'
+version = '0.2.0'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['835009ba6d623a36eee8056975d7cdbeebb0e0091a058b572ed433fb12ae18e8']
+
```

this should make the test suite happier
```suggestion

builddependencies = [('binutils', '2.35')]

```